rebasehelper/application.py

```diff
@@ -423,6 +425,8 @@ def prepare_sources(self):
                     if dest_dir:
                         Application.extract_sources(rest, os.path.join(self.execution_dir, sources_dir, dest_dir))
 
+        self.old_dir = old_dir
+        self.new_dir = new_dir
         self.run_package_checkers(self.results_dir, category="SOURCE")
```

Comment:
Why don't you just pass `old_dir` and `new_dir` into the method as kwargs?

Reply:
The question is still valid. The class variables seem totally unnecessary to me.